src/app/features/create-event/event-details-form/event-details-form.component.html

```diff
@@ -152,7 +159,49 @@ <h3 class="font-heading-3">Opis wydarzenia</h3>
         </mat-error>
         <mat-hint>Liczba znaków: {{ descriptionCtrl.value.length }}/4000 </mat-hint>
       </mat-form-field>
+
+      <mat-form-field class="details-form__chip-list" appearance="fill">
+        <mat-label>Hashtagi</mat-label>
+        <mat-chip-grid #chipGrid aria-label="Hashtag selection" formControlName="hashtags">
+          <mat-chip-row *ngFor="let hashtag of hashtags" (removed)="remove(hashtag)">
+            <span matChipAvatar>#</span>
+            {{ hashtag }}
+            <button matChipRemove [attr.aria-label]="'remove ' + hashtag">
+              <mat-icon>cancel</mat-icon>
+            </button>
+          </mat-chip-row>
+        </mat-chip-grid>
+        <mat-error *ngIf="hashtagsCtrl.errors as errors">
+          <ng-container *ngIf="errors['maxlength'] as maxLength; else hashtagsMinLength"
+            >Maks. liczba hashtagów: {{ maxLength.actualLength }}/{{ maxLength.requiredLength }}.</ng-container
+          >
+          <ng-template #hashtagsMinLength>
+            <ng-container *ngIf="errors['minSingleHashtagLength']; else hashtagsMaxLength"
+              >Hashtag musi mieć co najmniej 2 znaki</ng-container
+            >
+          </ng-template>
+          <ng-template #hashtagsMaxLength>
+            <ng-container *ngIf="errors['maxSingleHashtagLength']"
+              >Hashtag może mieć maksymalnie 20 znaków.</ng-container
```

Comment:
czy tych wartości `2` i `20` w komuniaktach o błędzie nie mamy zaszyte w obiekcie z błędem? jeśli tak to użyjmy ich zamiast hardkodować wartość